Ports/rubberband/package.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port='rubberband'
+version='3.3.0'
+depends=(
+    'libfftw3'
+    'libopus'
+    'libsamplerate'
+    'libsndfile'
+)
+useconfigure='true'
+configopts=(
+    '--cross-file' "${SERENITY_BUILD_DIR}/meson-cross-file.txt"
```

Comment:
```
   "--cross-file=${SERENITY_BUILD_DIR}/meson-cross-file.txt"
```

Reply:
This is the scheme of all other meson ports.

Reply:
That's not true - both the `lite-xl` and `wayland` ports use this style. Not sure why `glib`, `dosbox-staging` and `powdertoy` use a different style though; I thought we use `--long-style=value` in all our ports.